docs/topics/gpu.md

```diff
@@ -4,6 +4,18 @@ If you created a Kubernetes cluster with one or multiple Ubuntu node pools whose
 
 The NVIDIA drivers are automatically installed on every GPU agent in your cluster, so you don't need to do that manually, unless you require a specific version of the drivers. Currently, the installed driver is version 418.40.04.
 
+On Azure Stack Hub, however, NVIDIA drivers are not automatically installed. In this case the nvidia-device-plugin addon need to be explicitly enabled in apimodel.json, as follows:
```

Comment:
@jadarsie why are drivers not automatically installed for Azure Stack Hub?

Reply:
Not aware, it should be based on my understanding, will investigate.

Reply:
Looks like no one remembers why the nvidia driver functionality was explicitly disabled for Azure Stack?

Reply:
GPUs are relatively new, that's why it was not allowed, but I thought the ban was already lifted.